build.sbt

```diff
@@ -167,7 +170,7 @@ def aspectjProjectSettings = projectSettings ++ Seq(
 
 def basicSettings =  Defaults.itSettings ++ Seq(
   organization := "com.comcast.money",
-  version := "0.9.2-SNAPSHOT",
+  version := "0.10.9.0.1-SNAPSHOT",
```

Comment:
what kind of version number is this XD

Reply:
I wanted to indicate that it was Money `0.10.x` and that it was evolving with OpenTelemetry `0.9.1` but I botched that up and added an extra `0`.  I can remove that middle `0`, would you be more amenable to version `0.10.9.1`?

Reply:
Ahh, ok, I didn't catch the linkage to open telemetry.

We would need to move both thought.  For example, if we have a patch, we would be `0.10.1, 0.10.2` even if open tel stays `0.9.1`.

Also, unclear on how much change is going on in open tel land.  Like, if they release 0.9.2 and there are no substantive changes, would we do a new release just to stay in line with the version numbers?

This also begs the critical question, we need to update our release process to go through GHA and use github secrets so that anyone can push the button (different topic)

Reply:
The OTel Java project is currently versioning once a month.  They usually only rev the minor release, but this past month there was a problem and they had to cut an interim release, hence `0.9.1`.  This will require ongoing maintenance until OTel publishes a GA release and I am willing to keep up with any necessary changes.  These monthly releases have included breaking changes, which is why I thought it would be worthwhile to link the Money version to the OTel version to make it apparent how the versions are associated to each other.

I do agree that if Money needs to rev to keep up with the OTel API releases that simplifying/automating the release process would be a big help.

@ptravers You have been more involved with OTel than I have, what are your thoughts on their timeframe to GA?  I also asked in their [Gitter channel](https://gitter.im/open-telemetry/opentelemetry-java).

Reply:
They are currently racing to hit GA very soon. I would guess it's unlikely there are significant changes before GA. Java implementation is in clean up mode for GA

Reply:
Any suggestions on the version?  I thought it would be useful to tie the Money version to the OTel version to make it obvious which version of Money you'd want to integrate with existing OTel libraries, especially as OTel is still evolving towards GA.  It's somewhat unfortunate that Money and OTel are both currently 0.9.x as adopting OTel's version would not indicate that there are breaking changes.  If we feel that this version alignment is worthwhile we can hold on making a proper release until early November when OTel 0.10.0 (or 1.0?) is released, in which case we can then align versions.  But we'd still have to consider Money patches and whether we'd increment the patch version apart from OTel.

Or we could keep the two decoupled, rev Money to 0.10.x and move forward.  If keeping up with OTel involves breaking changes we rev Money, otherwise we cut a patch release.

Either way, IMO it would be good to stabilize Money and to be able to cut a 1.0 release alongside a 1.0 release of OTel.

Reply:
I think we go `0.10.0` and decouple from the OTel version, and I like the idea of doing a 1.0 release with OTel

Reply:
Cool beans, updated this PR to `0.10.0-SNAPSHOT`.

Reply:
that makes sense to me

Reply:
@HaloFour do you want to release the updates for W3C Trace support before this release? I would probably prefer to get those out as a patch release before releasing this as they don't need to be lumped into this breaking release.

Reply:
That's fine by me if you think that anyone would find it useful.

Reply:
I just thought that upstream legacy systems could be converted to a non money trace without re-instrumenting the software.

Reply:
I'll leave it to @pauljamescleary since he's the one with the keys to publish.